rdr_service/config.py

```diff
@@ -291,6 +291,8 @@
 PM_HEIGHT_CODES = 'pm_height_codes'
 PM_WEIGHT_CODES = 'pm_weight_codes'
 
+PARTICIPANT_CLEANUP_CC_LIST = 'participant_cleanup_cc_list'
```

Comment:
is this code constant needed? I am not sure if it is being pulled on L925 of rdr_service/offline/main.py